guac_core/src/storages/in_memory.rs

```diff
@@ -0,0 +1,100 @@
+use channel_client::types::{Channel, ChannelStatus};
+use clarity::Address;
+use crypto::CryptoService;
+use failure::Error;
+use futures::future::ok;
+use futures::Future;
+use num256::Uint256;
+use payment_contract::ChannelId;
+use qutex::Qutex;
+use rand;
+use rand::prelude::*;
+use rand::RngCore;
+use std::collections::HashMap;
+use storage::Storage;
+use CRYPTO;
+
+struct ChannelData {
+    url: String,
```

Comment:
Is it really `ChannelData` and not e.g. `PeerData`? The counterparty's URL has nothing to do with actual channels and therefore is not channel data.

Reply:
It was poor choice. I've just put url into Channel. ChannelData is removed already.